src/test/config.ts

```diff
@@ -3,6 +3,18 @@ export const config = {
   SMOKE_TEST_USER_USERNAME: process.env.SMOKE_TEST_USER_USERNAME,
   SMOKE_TEST_USER_PASSWORD: process.env.SMOKE_TEST_USER_PASSWORD,
   PASSWORD: 'Pa55word11',
+  staleUser: {
+    email: 'staleUser@test.com',
+    password: 'Pa55word11',
+    role: 'IDAM_SUPER_USER',
+    firstName: 'staleTest'
+  },
+  suspendUser: {
+    email: 'suspenduser@test.com',
+    password: 'Pa55word11',
+    role: 'IDAM_SUPER_USER',
+    firstName: 'suspendTest'
+  },
```

Comment:
Hi @ashwini-mv! Any reason to not have these in the testAccounts variable to keep all the test account data in the same place?